Compilation failure in branch v6-26-00-patches with gcc11, el8 and amd64

Hello,

We are trying to build root branch [v6-26-00-patches](https://github.com/root-project/root/compare/v6-26-00-patches), but it does not compile after commit 80c1ab6b6dab02c46d8cc66262686fd718b57ece. Here the stacktrace:
```
/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc11/lcg/root/6.26.11-214c85d364498a3e137a3783af2e7b69/root-6.26.11/gui/browsable/src/TDirectoryElement.cxx:184:16: error: 'std::string TDirectoryLevelIter::GetContent(const string&)' marked 'override', but does not override
  184 |    std::string GetContent(const std::string &kind) override
      |                ^~~~~~~~~~
/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc11/lcg/root/6.26.11-214c85d364498a3e137a3783af2e7b69/root-6.26.11/gui/browsable/src/TDirectoryElement.cxx: In member function 'std::string TDirectoryLevelIter::GetContent(const string&)':
/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc11/lcg/root/6.26.11-214c85d364498a3e137a3783af2e7b69/root-6.26.11/gui/browsable/src/TDirectoryElement.cxx:186:11: error: 'GetContentKind' was not declared in this scope; did you mean 'GetContent'?
  186 |       if (GetContentKind(kind) == kFileName)
      |           ^~~~~~~~~~~~~~
      |           GetContent
/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc11/lcg/root/6.26.11-214c85d364498a3e137a3783af2e7b69/root-6.26.11/gui/browsable/src/TDirectoryElement.cxx:186:35: error: 'kFileName' was not declared in this scope
  186 |       if (GetContentKind(kind) == kFileName)
      |                                   ^~~~~~~~~
/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc11/lcg/root/6.26.11-214c85d364498a3e137a3783af2e7b69/root-6.26.11/gui/browsable/src/TDirectoryElement.cxx:187:17: error: 'fFileName' was not declared in this scope
  187 |          return fFileName;
      |                 ^~~~~~~~~
```
It successfully build up to commit https://github.com/root-project/root/commit/58c6a86fbe840a0f7fdd95d710755e925300120f. We are targeting `el8`, `amd64` and `gcc11`. Could you please have a look?

Thanks,
Andrea.

FYI, @smuzaffar

--- a/gui/browsable/src/TDirectoryElement.cxx
+++ b/gui/browsable/src/TDirectoryElement.cxx
@@ -181,14 +181,6 @@ public:
    /** Returns full information for current element */
    std::shared_ptr<RElement> GetElement() override;
 
-   std::string GetContent(const std::string &kind) override
-   {
-      if (GetContentKind(kind) == kFileName)
-         return fFileName;
-
-      return ""s;
-   }
-
 };
 
 // ===============================================================================================================
@@ -433,6 +425,14 @@ public:
          return true;
       }
       return false;
+   }
+
+   std::string GetContent(const std::string &kind) override
+   {
+      if (GetContentKind(kind) == kFileName)
+         return fFileName;
+
+      return ""s;
    }
 
 };

